plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -469,4 +470,17 @@ public boolean isStorageSchemaSetWhenHidingIsEnabled()
     {
         return hideMaterializedViewStorageTable && materializedViewsStorageSchema.isPresent();
     }
+
+    public boolean isMetadataCacheEnabled()
+    {
+        return metadataCacheEnabled;
+    }
+
+    @Config("iceberg.metadata-cache.enabled")
+    @ConfigDescription("Enables in-memory caching of metadata files on coordinator")
```

Comment:
technically it won't be in-memory if alluxio is on, update comment?

Reply:
updated description and docs